pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -641,7 +640,10 @@ let
       NFS_V4_1              = yes;  # NFSv4.1 client support
       NFS_V4_2              = yes;
       NFS_V4_SECURITY_LABEL = yes;
-      NFS_LOCALIO           = whenAtLeast "6.12" yes;
+
+      # Fails with
+      # `fs/nfs/localio.o: relocation R_AARCH64_ADR_PREL_PG_HI21 against symbol `nfs_to' which may bind externally can not be used when making a shared object; recompile with -fPIC`
+      NFS_LOCALIO           = lib.mkIf (!stdenv.hostPlatform.isAarch64) (whenAtLeast "6.12" yes);
```

Comment:
This should probably be in a separate commit, since it's unrelated to Rust?  Otherwise LGTM.

Reply:
Done!